backend/director/agents/transcription.py

```diff
@@ -0,0 +1,101 @@
+import logging
+from director.agents.base import BaseAgent, AgentResponse, AgentStatus
+from director.core.session import TextContent, MsgStatus
+from director.tools.videodb_tool import VideoDBTool
+
+logger = logging.getLogger(__name__)
+
+class TranscriptionAgent(BaseAgent):
+    def __init__(self, session=None, **kwargs):
+        self.agent_name = "transcription"
+        self.description = (
+            "This is an agent to get transcripts of videos"
+        )
+        self.parameters = self.get_parameters()
+        super().__init__(session=session, **kwargs)
+
+    def run(self, collection_id: str, video_id: str, timestamp_mode: bool = False, time_range: int = 2) -> AgentResponse:
+        """
+        Transcribe a video and optionally format it with timestamps.
+
+        :param str collection_id: The collection_id where given video_id is available.
+        :param str video_id: The id of the video for which the transcription is required.
+        :param bool timestamp_mode: Whether to include timestamps in the transcript.
+        :param int time_range: Time range for grouping transcripts in minutes (default: 2 minutes).
+        :return: AgentResponse with the transcription result.
+        :rtype: AgentResponse
+        """
+        self.output_message.actions.append("Trying to get the video transcription...")
+        output_text_content = TextContent(
+            agent_name=self.agent_name,
+            status_message="Processing the transcription...",
```

Comment:
Could you please keep the ellipses (..) to a max of two in all status messages and actions?